keps/sig-instrumentation/0035-20190131-event-series.md

```diff
@@ -0,0 +1,59 @@
+---
+kep-number: 35
+title: Event series API
+authors:
+  - "@gmarek"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-instrumentation
+  - sig-scalability
+  - sig-architecture
+reviewers:
+  - "@wojtekt"
+  - "@bgrant0607"
+approvers:
+  - "@wojtekt"
+  - "@bgrant0607"
+editor: TBD
+creation-date: 2019-1-31
+last-updated: 2019-1-31
+status: implementable
+see-also:
+  - n/a
+replaces:
+  - n/a
+superseded-by:
+  - n/a
+---
+
+# Title
+
+Event series API
+
+## Table of Contents
+
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+
+
+This KEP wraps the already merged design doc under [kubernetes/community](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/events-redesign.md)
+
+## Graduation Criteria
+
+Beta:
+
+- noticeable performance improvement
```

Comment:
Compared to the current state of the new API, or compared to the original API?

Do we just expect no regression compared to the original API?

Reply:
I’d expect at least no regression in term of performance, but overall I’d expect this design to improve performance compared to the original API.

Reply:
I agree with Brian.